openedx_webhooks/github/dispatcher/rules/github_activity.py

```diff
@@ -21,6 +19,8 @@ def _find_issues_for_pull_request(pull_request_url, jira=jira):
     Returns:
         jira.client.ResultList[jira.Issue]
     """
+    if jira is None:
+        from ....lib.jira.client import jira
```

Comment:
The JIRA client relies on environment variables to be set, which may not be present during testing (and don't need to be, since we're not actually making calls to the JIRA server).

The problem is the JIRA client fails if it can't make a proper connection to the server.